README.md

```diff
@@ -43,120 +43,12 @@ linstorsatelliteconfigurations.piraeus.io/all-satellites created
-  If [cert-manager](https://cert-manager.io/) is available, the certificates can be created on demand by the operator.
-  ```yaml
-  apiVersion: piraeus.io/v1
-  kind: LinstorCluster
-  metadata:
-    name: linstorcluster
-  spec:
-    internalTLS:
-      secretName: linstor-controller-internal-tls
-      certManager:
-        kind: Issuer
-        name: piraeus-root
-  ```
-* A list of properties to apply on the controller level. For example, to configure LINSTOR to allocate Ports for DRBD
-  starting at 8000 (instead of the default 7000), you can apply the following change to the LinstorCluster resource
-  ```yaml
-  apiVersion: piraeus.io/v1
-  kind: LinstorCluster
-  metadata:
-    name: linstorcluster
-  spec:
-    properties:
-      - name: TcpPortAutoRange
-        value: "8000-8999"
-  ```
-* [Kustomize patches](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/patches/) to apply to
-  resources. When the operator applies resources, it will use `kustomize` to adapt the [base resources](./pkg/resources).
-
-  For example, to change the number of replicas for the CSI Controller deployment, you can use the following patch:
-  ```yaml
-  apiVersion: piraeus.io/v1
-  kind: LinstorCluster
-  metadata:
-    name: linstorcluster
-  spec:
-    patches:
-      - target:
-          kind: Deployment
-          name: csi-controller
-        patch: |-
-          apiVersion: apps/v1
-          kind: csi-controller
-          metadata:
-            name: csi-controller
-          spec:
-            replicas: 3
-  ```
-
-The `LinstorSatelliteConfiguration` resources allow customizing a set of satellites at once. The `spec.nodeSelector` is
-used to determine which customization is applied on a node.
-
-For example, if you have a set of storage nodes, all labelled with `example.com/storage-node=""`, and you want to:
-* Configure the default network interface to be the IPv6 address of your Pods.
-* Configure secured control plane traffic via TLS, using a [cert-manager](https://cert-manager.io/) issuer named
-  `piraeus-root`.
-* Configure a storage pool on all nodes, setting up on the devices `/dev/vdb` using a LVM thinpool `vg1/thin1` and naming
-  it `thin1` in LINSTOR.
-```
-apiVersion: piraeus.io/v1
-kind: LinstorSatelliteConfiguration
-metadata:
-  name: storage-satellites
-spec:
-  nodeSelector:
-    example.com/storage-node: ""
-  internalTLS:
-    certManager:
-      kind: Issuer
-      name: piraeus-root
-  properties:
-    - name: PrefNic
-      value: default-ipv6
-  storagePools:
-    - name: thin1
-      lvmThin:
-        volumeGroup: vg1
-        thinPool: thin1
-      source:
-        hostDevices:
-          - /dev/vdb
-```
+We are currently working on improving our documentation. Additional tutorials, how-to guides and explanation will be
+added soon.
+
+### [API Reference](./docs/reference)
+
+Lists all user settable options that control the state of Piraeus Datastore.
```

Comment:
This sentence is no longer quite right.

"settable" is also an unusual word, by the way. "configurable" or rephrased with "can be set" are more common.

Reply:
Do you have a suggestion? I can't really think of a nice summary.

Reply:
Based on your intro in the document itself:

> The API Reference for the Piraeus Operator. Contains documentation of the LINSTOR related resources that the user can modify or observe.